docs/terminal/brisbane.md

```diff
@@ -95,6 +95,38 @@ These standard runway assignments may be modified strategically by the BAS or BF
 !!! example
     **BAS:** “QFA660, Turn left heading 040, join final Runway 01L, from that heading Cleared Independent Visual Approach Runway 01L, not below 2400 until established on the PAPI”    
 
+## Helicopter Operations
```

Comment:
Not sure if the different headings will effect the terminal/brisbane.md changes made under the YBAF Helicopter Procedures